include/ui_components/ui_manager.hpp

```
@@ -18,7 +18,7 @@
 #include "tap_event.hpp"
 
 namespace Debug {
-inline bool ui_debug_mode = false;
+inline bool ui_debug_mode = true;
```

Clang (14.0.6)

[warning] variable 'ui_debug_mode' is non-const and globally accessible, consider making it const

(cppcoreguidelines-avoid-non-const-global-variables)